crates/sensors/src/ads1299/registers.rs

```diff
@@ -2,30 +2,39 @@
 
 // ADS1299 Register Value Constants
 pub const MUX_NORMAL: u8 = 0 << 0;
-pub const PD_REFBUF: u8 = 1 << 7;      // 1 : Enable internal reference buffer
-pub const BIAS_MEAS: u8 = 1 << 4;      // 1 : BIAS_IN signal is routed to the channel that has the MUX_Setting 010 (VREF)
-pub const BIASREF_INT: u8 = 1 << 3;    // 1 : BIASREF signal (AVDD + AVSS) / 2 generated internally
-pub const PD_BIAS: u8 = 1 << 2;        // 1 : BIAS buffer is enabled
+pub const PD_REFBUF: u8 = 1 << 7; // 1 : Enable internal reference buffer
+pub const BIAS_MEAS: u8 = 1 << 4; // 1 : BIAS_IN signal is routed to the channel that has the MUX_Setting 010 (VREF)
+pub const BIASREF_INT: u8 = 1 << 3; // 1 : BIASREF signal (AVDD + AVSS) / 2 generated internally
+pub const PD_BIAS: u8 = 1 << 2; // 1 : BIAS buffer is enabled
 pub const BIAS_LOFF_SENS: u8 = 1 << 1; // 1 : BIAS sense is enabled
-pub const SRB1: u8 = 1 << 5;           // 1 : Switches closed.. This bit connects the SRB1 to all 4, 6, or 8 channels inverting inputs
+pub const SRB1: u8 = 1 << 5; // 1 : Switches closed.. This bit connects the SRB1 to all 4, 6, or 8 channels inverting inputs
 pub const DC_TEST: u8 = 3 << 0;
 pub const POWER_OFF_CH: u8 = 0x81;
 pub const BIAS_SENS_OFF_MASK: u8 = 0x00;
 pub const DAISY_DISABLE: u8 = 1 << 6;
 
 // Register Addresses
-pub const REG_ID_ADDR    : u8 = 0x00;
-pub const CONFIG1_ADDR   : u8 = 0x01; pub const CONFIG1_REG: u8 = 0x90;
```

Comment:
we had it set up so it was `register address; register default value;` on the same line

i guess separate lines is fine